force-app/main/default/classes/ValidationCalloutTest.cls

```diff
@@ -0,0 +1,40 @@
+@isTest
+private class ValidationCalloutTest {
+    @isTest static void personCreationTest() {
+        Person__c p=TestDataFactory.createPeople('test', 1)[0];
+        p.Phone__c='+381604009911';
+
+        StaticResourceCalloutMock mock = new StaticResourceCalloutMock();
+        mock.setStaticResource('PhoneValidationSuccess');
+        mock.setStatusCode(200);
+        mock.setHeader('Content-Type', 'application/json;charset=UTF-8');
+        Test.setMock(HttpCalloutMock.class, mock);
+
+        Test.startTest();
+        insert p;
+        Test.stopTest();
+        //problem if another test creates more phone validation results??????
+        System.assert([select Name from Phone_Validation_Result__c where Person__c=:p.Id limit 1]!=null, 'The Phone Validation Result was not created');
+
+    }
+
+    @isTest static void personErrorUpdateTest() {
```

Comment:
Add the following tests
1) just fail scenario
2) multiple records
3) phone is empty/null/empty space
4) phone is populated but not changed